TEveEventDisplay/src/TEveMu2e_base_classes/TEveMu2eMainWindow.h

```diff
@@ -109,17 +130,18 @@ namespace mu2e{
       TEveMu2eMCInterface *pass_mc	= new TEveMu2eMCInterface(); 
       TEveMu2eProjectionInterface *pass_proj = new TEveMu2eProjectionInterface();
       
-      double texttime = -1;
+      // data options
+      DrawOptions DrawOpts;
       std::vector<double> *clusterenergy = 0;
```

Comment:
The preferred style is to use nullptr, not 0. The times that it matters are rare but just always do it correctly and you don't need to think about when it matters.

Reply:
sure, I am making that transition now, this =0 thing has caused us a lot of issues with seg faults later on, which is my I am changing to this DrawOpt, I should have updated the header too

Reply:
I don't have time to look at this carefully until next week but here is an initial observation.  I looked at how the variable clusterenergy is used inside TEveMu2eMainWindow.cc .    It looks like you never really use this a a pointer - you always copy into it.  So it should be an object, not a pointer to an object.

Reply:
ah. this has to be 0 since it does the whole max/min thing later on. I will make all the others nullptrs though